simapp/app.go

```diff
@@ -301,6 +301,11 @@ func NewSimApp(
 		BlockedAddresses(),
 		authtypes.NewModuleAddress(govtypes.ModuleName).String(),
 	)
+	app.BankKeeper.SetHooks(
```

Comment:
If this proposal is accepted, the app wiring configuration should be done for bank hooks.
You can look at staking:
- https://github.com/cosmos/cosmos-sdk/blob/v0.47.0-rc1/x/staking/module.go#L199
- https://github.com/cosmos/cosmos-sdk/blob/v0.47.0-rc1/x/staking/module.go#L245-L282
- https://github.com/cosmos/cosmos-sdk/blob/v0.47.0-rc1/proto/cosmos/staking/module/v1/module.proto#L13-L16

Reply:
Of course, this is why I mentioned:
> If this proposal is accepted
Don't do it already, just in case :)

Reply:
lmk when and if the team is looking to move forward with this ser. Have a good weekend sers